dataclients/kubernetes/kube.go

```diff
@@ -498,9 +502,21 @@ func (c *Client) LoadAll() ([]*eskip.Route, error) {
 	c.current = mapRoutes(r)
 	log.Debugf("all routes loaded and mapped")
 
+	go c.registerSigtermHandler()
```

Comment:
this will spawn a go routine for every test that never return. We should try to make the data client cleanup its resources (this is the first such resource). I have sg on my mind how, making a suggestion in a sub-branch.